app/lib/handle-input-change.js

```diff
@@ -0,0 +1,35 @@
+import DOMPurify from 'dompurify';
+
+export default function handleInputChange(e) {
+  /*
+  Using this module is a little odd.
+  Ensure that it's called in the context of a JSON-API Model instance.
+  */
+  if (typeof this.update !== 'function' || typeof this.emit !== 'function') {
+    throw new Error('Bind the handleInputChange function to a json-api-client Model instance')
+  }
+
+  let valueProperty = 'value';
+  switch (e.target.type) {
```

Comment:
just confirming that I think with `let valueProperty = 'value'`, then setting a `default` within the `switch` isn't necessary?

Reply:
A default might be a good idea. The `let` is here so that `valueProperty` is scoped and available outside the switch.